bridgev2/database/upgrades/21-disappearing-message-fkey.sqlite.sql

```diff
@@ -0,0 +1,30 @@
+-- v21: Add foreign key constraint from disappearing_message.mx_room to portals.mxid
+-- transaction: off
+PRAGMA foreign_keys = OFF;
```

Comment:
I don't think this migration requires disabling foreign keys or transactions

Also, portal will need a unique key on mxid (I think one can be added in sqlite without recreating the table by creating a unique index)

Reply:
Done and done!